Pygame simon

smiles1313 opened the pull request.

smiles1313 committed:
Add files via upload

The intro and displaying the buttons for Simon. (Need to have pygame to run)

smiles1313 committed:
Add files via upload

Problems with the flash button light and allowquit function.

smiles1313 committed:
Add files via upload

Problems with breaking out of loops, showing button flash (because of time), and quitting using quit() function.

getmagic-shilpa-gupta commented on Simon.py:
```diff
@@ -0,0 +1,282 @@
+bottom_mid = pygame.Rect(300, 250, 100, 100)
+#blue_rect  = pygame.draw.rect(DISPLAY, blue, bottom_mid)
+
+bottom_right = pygame.Rect(450, 250, 100, 100)
```
Right now the display is three squares in the bottom and one on the top of the middle one. Is that the layout you envisioned?

smiles1313 replied:
Yes

getmagic-shilpa-gupta commented on Simon.py:
```diff
@@ -0,0 +1,282 @@
+#K_RIGHT               right arrow
+#K_LEFT                left arrow
+
+def main():
```
Right now the boxes are flashing too quickly. Would recommend following a process
1. Can you start with the drawing board with black background and displaying the 4 boxes
2. Add to 1, click tracking on the boxes 
3. Add lighting to 1 box randomly

smiles1313 committed:
Add files via upload

Can click buttons 'Start' and 'Exit'.

smiles1313 committed:
Add files via upload

'Start' & 'Exit' buttons, 'Begin Playing' button, and can click the pattern on the screen as well as use arrow keys.

smiles1313 committed:
Add files via upload

Added game_quit() to more places, which stopped lag in testing main.

smiles1313 committed:
Add files via upload

smiles1313 committed:
Add files via upload

Only two buttons flash one color (yellow). Testing flash and color.

smiles1313 committed:
Add files via upload

Light flash is now visible for longer period of time (desired time).

smiles1313 committed:
Add files via upload

smiles1313 committed:
Update README.md

smiles1313 committed:
Add files via upload
